features/admin.organizations.v1/api/organization.ts

```diff
@@ -480,3 +481,52 @@ export const unshareApplication = (
         return Promise.reject(error?.response?.data);
     });
 };
+
+/**
+ * Get a list of organizations' meta attributes.
+ *
+ * @param filter - The filter query.
+ * @param limit - The maximum number of meta attributes to return.
+ * @param after - The previous range of data to be returned.
+ * @param before - The next range of data to be returned.
+ * @param recursive - Whether we need to do a recursive search.
+ * @param isRoot - Whether the organization is the root
+ *
+ * @returns a promise containing the response with the meta attributes.
+ */
+export const getOrganizationsMetaAttributes = (
+    filter: string,
+    limit: number,
+    after: string,
+    before: string,
+    recursive: boolean = false,
+    isRoot: boolean = false
+): Promise<OrganizationsMetaAttributesListInterface> => {
+    const config: HttpRequestConfig = {
+        headers: {
+            Accept: "application/json",
+            "Content-Type": "application/json"
+        },
+        method: "GET",
+        params: {
+            after,
+            before,
+            filter,
+            limit,
+            recursive
+        },
+        url: `${ isRoot
+            ? store.getState().config.endpoints.rootOrganization
+            : store.getState().config.endpoints.organizations }/organizations/meta-attributes`
+    };
+    return httpClient(config)
```

Comment:
Let's use the useRequest hook based approach for GET requests.

Ref: https://github.com/wso2/identity-apps/blob/05605b9367987c95ce06add63ce0e95494a90b2d/apps/myaccount/src/api/branding-preferences.ts#L42-L71